src/3d/qgs3dutils.h

```diff
@@ -60,6 +60,13 @@ class _3D_EXPORT Qgs3DUtils
      */
     static QImage captureSceneImage( QgsAbstract3DEngine &engine, Qgs3DMapScene *scene );
 
+    /**
+     * Captures the depth buffer of the current 3D scene of a 3D engine. The function waits
+     * until the scene is not fully loaded/updated before capturing the image.
```

Comment:
The second option is not very good since in most cases only one pixel from the depth buffer is needed.
I added a decode function and a note to use the decode function.